todoapp/users/models.py

```
@@ -1,30 +1,63 @@
+        auto_now_add=True
+    )
+    
+    is_active = models.BooleanField(default=True)
+    
+    USERNAME_FIELD = "email"
+    REQUIRED_FIELDS = ["first_name", "last_name"]
+    
     objects = UserManager()
-
+    
+    def __init__(self, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)